cm/README.md

```diff
@@ -3,6 +3,8 @@ Base directory for the calculation modules (cm).
 A calculation module is an asynchronous slow task (from seconds to hours of latency for the answer). 
 A calculation module takes a set of rasters, a selection vector shape and an optional set of parameters
 
+For basic example, see [example_empty](./example_empty) CM. 
+And for more detailed example, see [example_multiply](./example_multiply) CM.
```

Comment:
I would write "For a non implemented example..." (and add comments in the code to indicate the parts to be completed),  and "And for a simple example..."

Reply:
I made the suggested changes

Reply:
Y cannot see your changes, do you have to do another merge request? (I'm not used to work  whit github yet)

Reply:
I think you should go to the "Files changed" section to see the latest version of the branch. 
Unfortunately, I think we don't have the update in the conversation section, we just have the "outdated" label.

Reply:
Thank you!